test/strings/types.jl

```diff
@@ -174,11 +165,6 @@ for T in (String, GenericString)
                 s = convert(T, string(prefix, c, suffix))
                 r = reverse(s)
                 ri = search(r, c)
-                @test r == RevString(s)
```

Comment:
Hmm, you don't check that `r` is correct now. Should probably hardcode the results.

Reply:
We have more extensive tests for the correctness of `reverse` in test/strings/unicode.jl; I assume those would be sufficient.